server/src/main/java/org/bonitasoft/console/common/server/servlet/TenantFileUploadServlet.java

```diff
@@ -49,16 +47,8 @@ protected APISession getAPISession(final HttpServletRequest request) {
     }
 
     @Override
-    protected void checkUploadSize(final HttpServletRequest request, final FileItem item) throws FileTooBigException {
-        final long contentSize = item.getSize();
-        final long maxSize = getConsoleProperties(getAPISession(request).getTenantId()).getMaxSize();
-        if (contentSize > maxSize * 1048576) {
-            final String errorMessage = "file " + item.getName() + " too big !";
-            if (LOGGER.isLoggable(Level.SEVERE)) {
-                LOGGER.log(Level.SEVERE, errorMessage);
-            }
-            throw new FileTooBigException(errorMessage, item.getName(), String.valueOf(maxSize));
-        }
+    protected void setUploadSizeMax(final ServletFileUpload serviceFileUpload, final HttpServletRequest request) {
+        serviceFileUpload.setFileSizeMax(getConsoleProperties(getAPISession(request).getTenantId()).getMaxSize() * 1048576);
```

Comment:
1048576: Where this number come from?
You shoudl use a constant which will explained it